packages/auth-client/src/index.ts

```diff
@@ -329,15 +329,19 @@ export class AuthClient {
     // The event handler for processing events from the IdP.
     private _eventHandler?: (event: MessageEvent) => void,
   ) {
-    const logout = this.logout.bind(this);
-    const idleOptions = _createOptions?.idleOptions;
+    this._registerDefaultIdleCallback();
+  }
+
+  private _registerDefaultIdleCallback() {
+    const idleOptions = this._createOptions?.idleOptions;
     /**
      * Default behavior is to clear stored identity and reload the page.
      * By either setting the disableDefaultIdleCallback flag or passing in a custom idle callback, we will ignore this config
      */
     if (!idleOptions?.onIdle && !idleOptions?.disableDefaultIdleCallback) {
       this.idleManager?.registerCallback(() => {
-        logout();
+        console.log('User is idle. Logging out.');
```

Comment:
Since you can register custom callbacks to the `idleManager`, I'd prefer to leave any `console.log` statements up to the dev

```suggestion
```